.drone.yml

```diff
@@ -8,7 +8,21 @@ pipeline:
       - apk add --no-cache bash sed grep bc coreutils curl
       - ./ci-build.sh
     when:
-      event: [push, pull_request]
+      event: push
+
+  scan-clamav:
+    image: quay.io/ukhomeofficedigital/anchore-submission:latest
```

Comment:
Not using tagged version?

Reply:
I think that's very common in drone plugins (http://plugins.drone.io/drone-plugins/) ...